api/src/main/java/fade/mirror/filter/by/Annotations.java

```diff
@@ -12,10 +12,18 @@
 
     @NotNull R withAnnotations(@NotNull List<Class<? extends Annotation>> annotations, @NotNull RewriteOperation operation);
 
-    @NotNull R withAnnotations(@NotNull List<Class<? extends Annotation>> annotations);
-
-    @NotNull R withAnnotation(@NotNull Class<? extends Annotation> annotation, @NotNull RewriteOperation operation);
-
-    @NotNull R withAnnotation(@NotNull Class<? extends Annotation> annotation);
-
+    @NotNull
```

Comment:
`@NotNull` should be placed prior to the typename, otherwise it would be a method annotation. Same with methods below.